.github/pull_request_template.md

```diff
@@ -8,19 +8,38 @@ Please click the `Preview` tab and select a different PR template if your change
 ## Describe your changes (can list changeset entries if preferable)
 
 
-## Author Checklist (complete before requesting a review)
-- [ ] I have performed a self-review of my code
-- [ ] I have added thorough tests where applicable (unit / component / visual)
-- [ ] I have reviewed the `PIE Storybook`/`PIE Docs` PR preview
-- [ ] I have reviewed visual test updates properly before approving
+## Author Checklist (complete before requesting a review, do not delete any)
+- [ ] I have performed a self-review of my code.
+- [ ] I have added thorough tests where applicable (unit / component / visual).
+- [ ] I have reviewed the `PIE Storybook`/`PIE Docs` PR preview.
+- [ ] I have reviewed visual test updates properly before approving.
 - [ ] If changes will affect consumers of the package, I have created a changeset entry.
-- [ ] If a changeset file has been created, I have used the `/snapit` functionality to test my changes in a consuming application
+- [ ] If a changeset file has been created, I have tested these changes in [PIE Aperture](https://github.com/justeattakeaway/pie-aperture/) using the `/test-aperture` command.
+
+## Not-applicable Checklist items
+Please move any Author checklist items that do not apply to this pull request here.
+
+---
+
+## Testing
+[How do I test my changes?](https://github.com/justeattakeaway/pie/wiki/PIE-Aperture)
+
+| Task                   | Link                             |
+|------------------------|----------------------------------|
+| Aperture PR            | [🔗](#) |
+| NextJS 14 deployment   | [🔗](#) |
+| Nuxt 3 deployment      | [🔗](#) |
+| Vanilla deployment     | [🔗](#) |
```

Comment:
Not sure if we should bother putting the deployment links here. I think we discussed a while back that we should probably update the `test-aperture` command to provide feedback once complete to provide these links

Reply:
I can see your point - the aim of this was to make it far more intentional and less passive on the author's side. How about we add them for now and revisit once we've done the changes you suggest? Would love to pair on it! 👍